modules/Kconfig.mcuboot

```diff
@@ -62,6 +62,13 @@ config MCUBOOT_SIGNATURE_KEY_FILE
 
 	  If left empty, you must sign the Zephyr binaries manually.
 
+config MCUBOOT_SIGNATURE_TYPE_PURE
+	bool "Verify signature over the image instead of image hash"
+	depends on MCUBOOT_ENCRYPTION_KEY_FILE = ""
```

Comment:
Not sure of the relation to encryption

Reply:
Yeah, so there is no precedence for that. What previously has been done is that signature was on unencrypted image, which means that first you have to decrypt images by chunks, calculate sha and then check signature of that sha.
With Pure it is not possible, at this point, to verify signature because the whole image is not available till written to storage.

Reply:
Seems I misread the `=` as `!=`, change to `depends on !BOOT_ENCRYPTION`